docs/SUMMARY.md

```diff
@@ -3,27 +3,28 @@
 * [Welcome](book_files/csi_block_storage_kc_welcome.md)
 * [What's new](book_files/csi_block_storage_kc_whatsnew.md)
 * [Release notes](book_files/csi_block_storage_kc_rn.md)
-	* [What's new in 1.11.0](content/release_notes/whats_new.md)
-	* [Compatibility and requirements](content/release_notes/compatibility_requirements.md)
-		* [Supported storage systems](content/release_notes/supported_storage.md)
-		* [Supported operating systems](content/release_notes/supported_os.md)
-		* [Supported orchestration platforms](content/release_notes/supported_orchestration.md)
-	* [Change log](content/release_notes/change_log.md)
```

Comment:
these versions table looks horizontal instead of vertical in the view of the file.